builds/e2e/templates/nested-create-identity.yaml

```diff
@@ -16,7 +16,6 @@ steps:
         iotHubName=$(echo $tmp | sed -n 's/\(.?*\)\..*/\1/p')
         echo "Found Hub name: ${iotHubName}"
 
-        az account set --subscription $(azure.subscription)
```

Comment:
This line was deleted because it is unnecessary; this script was moved inside an Azure CLI task some time ago, and the task already logs into Azure and sets the suscription for you.